flink-connector/flink-connector-gcp-pubsub/src/test/java/com/google/pubsub/flink/PubSubSourceTest.java

```diff
@@ -58,6 +58,36 @@ public void build_invalidSchema() throws Exception {
     assertThrows(IllegalStateException.class, builder::build);
   }
 
+  @Test
+  public void build_invalidMaxOutstandingMessagesCount() throws Exception {
+    assertThrows(
+        NullPointerException.class,
+        () -> PubSubSource.<String>builder().setMaxOutstandingMessagesCount(null));
+    PubSubSource.Builder<String> builder =
```

Comment:
this feels like two different tests to me. one for NullPointerException and one for IllegalArgumentException. (same below)

Reply:
Created separate test cases